packages/components/psammead-navigation/README.md

```diff
@@ -71,6 +122,76 @@ import { latin } from '@bbc/gel-foundations/scripts';
 </Navigation>;
 ```
 
+## Canonical Dropdown Navigation Usage
+
+```jsx
+import React from 'react';
+import { CanonicalDropdown, DropdownUl, DropdownNavigationLi } from '@bbc/psammead-navigation/dropdown-navigation';
+import { latin } from '@bbc/gel-foundations/scripts';
+
+<CanonicalDropdown announcedText="Menu" onClose={() => {}}>
+  <DropdownUl>
+    <DropdownNavigationLi
+        script={latin}
+        service="news"
+        key="Home"
+        url="/"
+        active
+        currentPageText="Current page"
+    >
+      Home
+    </DropdownNavigationLi>
+    <DropdownNavigationLi script={latin} service="news" key="Sport" url="/sport">
+      Sport
+    </DropdownNavigationLi>
+  </DropdownUl>
+</CanonicalDropdown>
+```
+
+## Amp Dropdown Navigation Usage
+
+```jsx
+import React from 'react';
+import { AmpDropdown, DropdownUl, DropdownNavigationLi } from '@bbc/psammead-navigation/dropdown-navigation';
```

Comment:
Could this just be `DropdownLi` so the naming convention is the same for both of these components?